src/osvr/Common/ImagingComponent.cpp

```diff
@@ -71,9 +67,7 @@ namespace common {
 
             template <typename T>
             void allocateBuffer(T &, size_t bytes, std::true_type const &) {
-                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(
-                                   cv::fastMalloc(bytes)),
-                               &cv::fastFree);
+                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(malloc(bytes)));
```

Comment:
If you could point me to those aligned allocation functions I'll implement them here.